Fix Github Actions CI Script (#485)
astronomer/docs: astro/ci-cd.md

   - `ASTRONOMER_KEY_ID` = `<your-key-id>`
   - `ASTRONOMER_KEY_SECRET` = `<your-key-secret>`

>    - `DEPLOYMENT_ID` = `<your-astro-deployment-id>`

Reviewer: ```suggestion
   - `ASTRONOMER_DEPLOYMENT_ID` = `<your-astro-deployment-id>`
```

> 2. Add the following to a new file in .github/workflows, for example `cicd.yaml`:

Reviewer 2: ```suggestion
2. Create a new YAML file in `.github/workflows` that includes the following configuration:
```

        runs-on: ubuntu-latest
        steps:
        - name: checkout repo
          uses: actions/checkout@v2.3.4
        - name: Deploy to Astro
          run: |
            brew install astronomer/cloud/astrocloud

>             astrocloud deploy ${{ secrets.DEPLOYMENT_ID }}

Reviewer: ```suggestion
            astrocloud deploy ${{ secrets.ASTRONOMER_DEPLOYMENT_ID }}
```
